products/mssqlserver.md

```diff
@@ -11,135 +11,415 @@ versionCommand: select @@version
 releasePolicyLink: https://learn.microsoft.com/lifecycle/products/?terms=SQL%20Server
 
 activeSupportColumn: true
+releaseColumn: true
+releaseColumnLabel: Latest CU
 releaseDateColumn: true
 extendedSupportColumn: Extended Security Updates
 
+customColumns:
+  - property: latestGdr
+    position: before-latest-column
+    label: Latest GDR
+    description: Latest General Distribution Release (GDR) versions
+
+# releaseDate/support/eol/extendedSupport: https://learn.microsoft.com/lifecycle/products/?terms=SQL%20Server
+# Overview over all versions: https://learn.microsoft.com/troubleshoot/sql/releases/download-and-install-latest-updates
 releases:
 -   releaseCycle: "2022"
     codename: Dallas
-    releaseLabel: "2022 CU9"
+    releaseLabel: "2022"
     releaseDate: 2022-11-16
     support: 2028-01-11
     eol: 2033-01-11
     extendedSupport: false
-    latest: "16.0.4085.2"
+    latestGdr: "16.0.1105.1"
+    latestGdrLink: https://support.microsoft.com/help/5029379
```

Comment:
Preview: https://deploy-preview-3992--endoflife-date.netlify.app/mssqlserver
I wanted to have a link on the custom column entry.
I didn't find a way in https://endoflife.date/contribute
The table is too wide for the web view and the user has to scroll to get to the Latest CU column.
@marcwrobel any thoughts?

Reply:
To keep this MR simpler I have removed the custom column and I'll file a separate MR for that.